src/test/groovy/FindOldestSupportedVersionStepTests.groovy

```diff
@@ -61,21 +61,28 @@ class FindOldestSupportedVersionStepTests extends ApmBasePipelineTest {
   void test_match() throws Exception {
     def ret = script.call(versionCondition: "^7.14.0")
     printCallStack()
-    assertTrue(ret.equals('7.14.0'))
+    assert ret.equals('7.14.0')
+  }
+
+  @Test
+  void test_match_ge() throws Exception {
+    def ret = script.call(versionCondition: ">=7.14.0")
+    printCallStack()
+    assert ret.equals('7.14.0')
   }
 
   @Test
   void test_snapshot() throws Exception {
     def ret = script.call(versionCondition: "^7.14.1")
     printCallStack()
-    assertTrue(ret.equals('7.14.1-SNAPSHOT'))
+    assert ret.equals('7.14.1-SNAPSHOT')
```

Comment:
OOC, What's the reason for using `assert` rather than `assertTrue`?

Reply:
`assert` shows the values implied in the assertion, while `assertTrue` shows only an empty error.

Error with `assertTrue` when the assertion fails:
```
[ERROR] FindOldestSupportedVersionStepTests.test_or_condition  Time elapsed: 1.943 s  <<< FAILURE!
java.lang.AssertionError
	at FindOldestSupportedVersionStepTests.test_or_condition(FindOldestSupportedVersionStepTests.groovy:118)
```
And when `assert` fails:
```
[ERROR] FindOldestSupportedVersionStepTests.test_or_condition_reversed  Time elapsed: 0.123 s  <<< FAILURE!
org.codehaus.groovy.runtime.powerassert.PowerAssertionError: 
assert ret.equals('7.14.1')
       |   |
       |   false
       7.14.0
	at FindOldestSupportedVersionStepTests.test_or_condition_reversed(FindOldestSupportedVersionStepTests.groovy:132)
```